R/g_boxplot.R

```diff
@@ -10,35 +10,35 @@
 #' of the box indicate the degree of dispersion (spread) and skewness in the
 #' data, and show outliers. In addition to the points themselves, they allow one
 #' to visually estimate various L-estimators, notably the interquartile range,
-#' midhinge, range, mid-range, and trimean.
+#' `midhinge`, range, mid-range, and `trimean`.
```

Comment:
Either backtick all estimators or none.
Would these actually not belong in the WORDLIST?